tests/components/samsungtv/test_media_player.py

```diff
@@ -1098,7 +1098,6 @@ async def test_play_media_invalid_type(hass: HomeAssistant) -> None:
         # only update called
         assert remote.control.call_count == 0
         assert remote.close.call_count == 0
-        assert remote.call_count == 1
```

Comment:
Note: with the coordinator, there is no automatic "update entity" anymore, and the update doesn't get called right away anymore.

https://github.com/home-assistant/core/blob/bfffcc3ad640fd25aae2fe28631a16349857bbe6/homeassistant/helpers/service.py#L978

Reply:
If we want to auto refresh after a service call we could borrow something like this
https://github.com/home-assistant/core/blob/c2b3bf3fb969300c011349ed7e63d7a4eaced732/homeassistant/components/tplink/entity.py#L24

I'm not sure its needed though